src/aosm/azext_aosm/inputs/helm_chart_input.py

```diff
@@ -245,9 +253,9 @@ def get_schema(self) -> Dict[str, Any]:
 
             if not schema:
                 # Otherwise, generate a schema from the default values in values.yaml.
-                logger.debug("Generating schema from values.yaml")
+                logger.debug("Generating schema from default values")
```

Comment:
Is this always the default values? or is it sometimes from the valuers.yaml? Maybe reword